core/designsystem/src/main/kotlin/com/dhc/designsystem/calendar/ui/DhcCalendarDay.kt

```diff
@@ -0,0 +1,70 @@
+package com.dhc.designsystem.calendar.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import com.dhc.designsystem.DhcTheme
+import com.dhc.designsystem.calendar.model.DhcCalendarDayUiModel
+
+@Composable
+fun DhcCalendarDay(
+    day: Int,
+    modifier: Modifier = Modifier,
+    uiModel: DhcCalendarDayUiModel = DhcCalendarDayUiModel.DidNotMission,
+) {
+    Box(
+        modifier = modifier
+            .border(
+                width = if (uiModel.borderColor != null) 2.dp else 0.dp,
+                color = uiModel.borderColor ?: androidx.compose.ui.graphics.Color.Transparent,
```

Comment:
```suggestion
                color = uiModel.borderColor ?: Color.Transparent,
```
P2 :  `androidx.compose.ui.graphics.Color` 을 import 를 통해 가져와서 full path 를 피해보는건 어떨까용